README.md

```diff
@@ -40,11 +40,18 @@ package main
 import "github.com/adampointer/cali"
 
 func main() {
-	cli := cali.Cli("cali")
```

Comment:
I think all the changes to this file correspond to my changes in https://github.com/LMHD/lucli/commit/6aab119501341fe4bd519e1c71fbeb11174c1a05 but I haven't double-checked. I'll do that later with a fresh pair of eyes.

Reply:
All good.